experiments/tester/src/RNTester/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { ScreenRect, Text, View } from 'react-native';
+import { Button, Callout, Separator } from '@fluentui/react-native';
+import { fabricTesterStyles } from '../Common/styles';
+
+export const CalloutTest: React.FunctionComponent<{}> = () => {
+  const [state, setState] = React.useState({
```

Comment:
It will make the code cleaner or easier to read if you use separate states for each callout. 

```
const [showStandardCallout, setShowStandardCallout] = React.useState(false);
const [showCustomizedCallout, setShowCustomizedCallout] = React.useState(false);
```